controllers/FollowController.js

```diff
@@ -49,8 +49,38 @@ const getFollowStatus = async (req, res) => {
   }
 };
 
+const getFollowerList = async (req, res) => {
```

Comment:
I renamed this to `getFollowerList`, so please make sure you update the front end to that